bundles/README.md

```diff
@@ -1,3 +1,3 @@
 # Overview
 
-This folder contains kyma bundle repositories. Every folder creates a repository available in github pages with the URL: `https://kyma-project.github.io/kyma-bundles/{FOLDER NAME}/`
+This folder contains kyma bundle repositories. Every folder creates a repository available in github pages with the URL: `https://kyma-project.github.io/bundles/{FOLDER NAME}/`
```

Comment:
1. kyma bundle repositories. -> `Kyma repositories with bundles.`
2. `GitHub Pages`
3. ``with the `https://kyma-project.github.io/bundles/{FOLDER NAME}/` URL.``